source/content/sftp.md

```diff
@@ -101,77 +106,83 @@ SFTP mode works with any standards-compliant SFTP client, including many GUI too
 
 A simple way to test for outbound network restrictions is to load a special web site that listens on all ports. To test your access outbound on port `2222`, try to load this web page in your web browser:
 
-* [http://portquiz.net:2222/](http://portquiz.net:2222/)
+- [http://portquiz.net:2222/](http://portquiz.net:2222/)
 
 If you **cannot** access that web page then your network or firewall is likely preventing you from accessing port `2222` outbound. Contact your network administrators to allow outbound access on port `2222`. Advanced users may also be comfortable [establishing an SSH tunnel through another server](/port-2222) instead.
 
 If you **can** access that web page on port `2222` then your issue does not appear to be network or firewall related.  Be sure to double-check or re-enter your [SFTP settings](/sftp/#sftp-connection-information), including SFTP mode and port `2222`. Contact Pantheon Support if you still have trouble.
 
 ### I registered my Pantheon account via Google. How do I connect to SFTP?
+
 We recommend [adding an SSH Key](/ssh-keys), which allows more security than a simple password. If you've registered via social login (Connect with Google) and you'd still like to add a password to your account, logout and visit [https://dashboard.pantheon.io/reset-password](https://dashboard.pantheon.io/reset-password)
 
 ### I can't write to my codebase on Test or Live.
 
 This is by design. Please see [Using the Pantheon Workflow](/pantheon-workflow#understanding-write-permissions-in-test-and-live) to learn why.
 
-
 ### SFTP changes do not show up in the Site Dashboard.
+
 Uncommitted SFTP changes may not be recognized by the Dev environment when the Site Dashboard is open in multiple tabs or windows. Close all windows and tabs then access the Site Dashboard in a single tab to resolve.
 
-### How do I find my site's Binding path?
+### How do I find my site's remote path?
+
 <Alert title="Note" type="info">
 
 You should *not* manually set the "Remote Path" in your SFTP client's settings, as this path changes from time to time due to the platform architecture. It is strongly recommended that you leave the Remote Path blank, and you will automatically be redirected to the proper directory when logging in.
 
 </Alert>
 
-You can find the Remote Path after [connecting to SFTP](#sftp-connection-information) via command line, using the `pwd` command:
+You can find the Remote Path, also known as the **binding path**, after [connecting to SFTP](#sftp-connection-information) via command line, using the `pwd` (print working directory) command:
 
-```
+```bash
 sftp> pwd
 Response: Remote working directory: /srv/bindings/daa068ccf4f8414596cddf5xxxxx
 ```
```

Comment:
@sparklingrobots this is the section of changes that closes #5472, humbly submitted for your review.

Reply:
This looks great; no changes.  Thank you! 👍